samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor

```diff
@@ -0,0 +1,40 @@
+﻿@page "/DataGridCellActivation"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    
```

Comment:
can you add a combobox to change ActivationMode property on the DataGrid?

Reply:
we can revisit this, none of the other platforms don't have one.